### ISSUE TITLE
Consider adopting the `x264` crate.

### ISSUE THREAD

**Issue by quadrupleslap** [rust-av/x264-rs#6]
I'm not very familiar with `x264`, but I needed it for a project I was working on, so I made a high-level wrapper of it [over here](https://github.com/quadrupleslap/x264) ([docs](https://docs.rs/x264)). If you think the crate is a decent enough *safe abstraction*, would you mind adopting it? Thanks for considering it!

**Comment by quadrupleslap** [comment]
Bump? It now works without libstd, and supports UYVY/YUYV too!

**Comment by lu-zero** [comment]
Since this repo existed before your hand-made version, why you didn't contribute to it instead?

**Comment by quadrupleslap** [comment]
It wasn't on crates.io, so I didn't know it existed! :P

**Comment by lu-zero** [comment]
I need some time to read all your work, I'm quite sure we could try to deduplicate at least some of the effort, I would keep the sys crate since the automated bindings would help who would like to build different abstractions.

**Comment by quadrupleslap** [comment]
Yeah, no problem, I'm using the x264-sys crate, too - if the build script misled you, that's just for detecting UYVY support, it doesn't do any linking or bindings generation.

**Comment by lu-zero** [comment]
Oh wow, I got really sidetracked, I'm going to publish a new version of the `-sys` crate soon :)
